### PR TITLE
fix: revert pypy 3.7 to 7.3.7

### PR DESCRIPTION
**Pull request by henryiii**
See #1031

### CODE REVIEW

**Comment by henryiii** [comment]
For consistency, how about just rolling it back to consistently be 7.3.7 everywhere until manylinux and the issue with PyPy are fixed (probably in a 7.3.9)?

File: cibuildwheel/resources/build-platforms.toml [hunk 1]
```diff
@@ -71,7 +71,7 @@ python_configurations = [
   { identifier = "cp310-macosx_x86_64", version = "3.10", url = "https://www.python.org/ftp/python/3.10.2/python-3.10.2-macos11.pkg" },
   { identifier = "cp310-macosx_arm64", version = "3.10", url = "https://www.python.org/ftp/python/3.10.2/python-3.10.2-macos11.pkg" },
   { identifier = "cp310-macosx_universal2", version = "3.10", url = "https://www.python.org/ftp/python/3.10.2/python-3.10.2-macos11.pkg" },
-  { identifier = "pp37-macosx_x86_64", version = "3.7", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.8-osx64.tar.bz2" },
+  { identifier = "pp37-macosx_x86_64", version = "3.7", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.7-osx64.tar.bz2" },
   { identifier = "pp38-macosx_x86_64", version = "3.8", url = "https://downloads.python.org/pypy/pypy3.8-v7.3.8-osx64.tar.bz2" },
```
Comment (henryiii) [inline cibuildwheel/resources/build-platforms.toml:75]:
```suggestion
  { identifier = "pp38-macosx_x86_64", version = "3.8", url = "https://downloads.python.org/pypy/pypy3.8-v7.3.7-osx64.tar.bz2" },
```

File: cibuildwheel/resources/build-platforms.toml [hunk 2]
```diff
@@ -89,6 +89,6 @@ python_configurations = [
   { identifier = "cp310-win_amd64", version = "3.10.2", arch = "64" },
   { identifier = "cp39-win_arm64", version = "3.9.10", arch = "ARM64" },
   { identifier = "cp310-win_arm64", version = "3.10.2", arch = "ARM64" },
-  { identifier = "pp37-win_amd64", version = "3.7", arch = "64", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.8-win64.zip" },
+  { identifier = "pp37-win_amd64", version = "3.7", arch = "64", url = "https://downloads.python.org/pypy/pypy3.7-v7.3.7-win64.zip" },
   { identifier = "pp38-win_amd64", version = "3.8", arch = "64", url = "https://downloads.python.org/pypy/pypy3.8-v7.3.8-win64.zip" },
```
Comment (henryiii) [inline cibuildwheel/resources/build-platforms.toml:93]:
```suggestion
  { identifier = "pp38-win_amd64", version = "3.8", arch = "64", url = "https://downloads.python.org/pypy/pypy3.8-v7.3.7-win64.zip" },
```

**Comment by henryiii** [comment]
Update: no, it looks like 3.9 is ready to go, so we could do 7.3.7 for 3.7, and 7.3.8 for 3.8 & 3.9. That would be fine.

**Comment by mayeut** [comment]
@henryiii, manylinux is only rolling back 3.7 to 7.3.7: https://github.com/pypa/manylinux/pull/1289

**Comment by henryiii** [comment]
Happy to follow manylinux. We'll need a bump to the new manylinux images, so maybe we should just do everything in #1031.

**Comment by mayeut** [comment]
> We'll need a bump to the new manylinux images, so maybe we should just do everything in https://github.com/pypa/cibuildwheel/pull/1031.

I was just about to propose that !

**Pull request closed without merging**